brave/brave-core
# Removes p2p

Resolves https://github.com/brave/brave-browser/issues/9862

PR that introduced this code: https://github.com/brave/brave-core/pull/5171

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- enable rewards
- connect uphold account
- go to rewards page
- make sure that you don't see p2p card

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
